src/components/input/index.jsx

```diff
@@ -295,6 +302,7 @@ Input.propTypes = {
 Input.defaultProps = {
     autoFocus: false,
     disabled: false,
+    placeholder: '',
```

Comment:
Let's not have a default for it since throwing undefined in the jsx tag has the same behaviour